versions/1.0.0.md

```diff
@@ -74,7 +73,7 @@ The list of actions MUST be applied in sequential order to ensure a consistent o
 
 The `extends` property can be used to indicate that the Overlay was designed to update a specific [[OpenAPI]] document. Where no `extends` is provided it is the responsibility of tooling to apply the Overlay document to the appropriate OpenAPI document(s).
 
-In the following example the `extends` property specifies that the overlay is designed to update the OpenAPI Tic Tac Toe example document using an absolute URL.
+In the following example the `extends` property specifies that the overlay is designed to update the OpenAPI Tic Tac Toe example document, identified by an absolute URI reference.
```

Comment:
You can just say "absolute URI" here as an absolute URI is by definition not a relative reference.  "absolute URI reference" is still technically correct (since "URI reference" encompasses all other URI terms) but sounds weird to me.

I don't feel strongly about this so feel free to leave it.

```suggestion
In the following example the `extends` property specifies that the overlay is designed to update the OpenAPI Tic Tac Toe example document, identified by an absolute URI.
```